java-util/src/main/java/io/druid/java/util/common/CompressionUtils.java

```diff
@@ -234,7 +236,7 @@ public InputStream openStream() throws IOException
                     return new BufferedInputStream(zipFile.getInputStream(entry));
                   }
                 },
-                new File(outDir, entry.getName()),
+                outFile,
```

Comment:
@bolkedebruin this is good intention, but won't this poison the page cache anyways?

Reply:
Yes, so a follow up PR I have planned - I mentioned it above - should also skip the cache for local copying. Please note that it would poison the cache twice (ie local copy, and then unzip).